tck/src/main/java/ee/jakarta/tck/concurrent/spec/ManagedThreadFactory/resourcedef/ManagedThreadFactoryDefinitionTests.java

```diff
@@ -35,15 +40,38 @@
 	@ArquillianResource(ManagedThreadFactoryDefinitionServlet.class)
 	URL baseURL;
 	
+	@ArquillianResource(ManagedThreadFactoryDefinitionOnEJBServlet.class)
+	URL ejbContextURL;
+	
 	@Deployment(name="ManagedThreadFactoryDefinitionTests", testable=false)
-	public static WebArchive createDeployment() {
-		return ShrinkWrap.create(WebArchive.class)
-				.addPackages(false, ManagedThreadFactoryDefinitionTests.class.getPackage(),
+	public static EnterpriseArchive createDeployment() {
+		
+		WebArchive war = ShrinkWrap.create(WebArchive.class, "ManagedThreadFactoryDefinitionTests_web.war")
+				.addPackages(false,
 						getFrameworkPackage(), 
 						getContextPackage(),
 						getContextProvidersPackage())
-				.addClasses(ContextServiceDefinitionServlet.class)
+				.addClasses(
+						ManagedThreadFactoryDefinitionOnEJBServlet.class,
+						ManagedThreadFactoryDefinitionServlet.class,
+						ContextServiceDefinitionServlet.class)
+//				.addAsWebInfResource(ContextPropagationTests.class.getPackage(), "web.xml", "web.xml")
 				.addAsServiceProvider(ThreadContextProvider.class.getName(), IntContextProvider.class.getName(), StringContextProvider.class.getName());
+		
+		JavaArchive jar = ShrinkWrap.create(JavaArchive.class, "ManagedThreadFactoryDefinitionTests_ejb.jar")
+				.addPackages(false, getFrameworkPackage(), ManagedThreadFactoryDefinitionTests.class.getPackage())
+				.deleteClasses(
+						ManagedThreadFactoryDefinitionOnEJBServlet.class,
+						ManagedThreadFactoryDefinitionServlet.class)
+				.addClasses(
+						ContextServiceDefinitionInterface.class,
+						ContextServiceDefinitionBean.class);
+//				.addAsManifestResource(ManagedScheduledExecutorDefinitionTests.class.getPackage(), "ejb-jar.xml", "ejb-jar.xml");
+//				TODO document how users can dynamically inject vendor specific deployment descriptors into this archive
```

Comment:
Remove this TODO if unneeded

Reply:
This is still needed per Kyle's other comment.